### ISSUE TITLE
Delete a goal? 

### ISSUE THREAD

**Issue by farmanp** [codito/habito#13]
Is there a way to delete a goal? I wrote your example in but I don't want to keep that: 

`habito delete writing` - is there something of this sort?

**Comment by codito** [comment]
This is available in master. Please try with `pip install git+https://github.com/codito/habito.git`.

```shell
# Get the id with `habito list`
> habito delete 1                                                                                                                                              
Are you sure you want to delete habit 1: writing (this cannot be undone!) [y/N]: N                                                                             
Habit 1: writing has not been deleted! 
```

**Comment by codito** [comment]
Let me know if it works for you. We can do a pip release in next few days (after a bit of testing).

**Comment by codito** [comment]
Fixed in `v1.0a5` version on pypi. Please upgrade and try it.
